index.html

```diff
@@ -8,8 +8,39 @@
 <div class="block">
   <h1><span class="header-inside">Projects</span></h1>
   <div id="projects-panel">
+    <table class="projects">
+      <noscript>
+        {% for p in site.data.projects %}
+        {% assign project = p[1] %}
```

Comment:
This is how `site.data.projects` is structured – it doesn't work without line 14. There might be more elegant ways to iterate but I didn't find... I'm not really sure why the data is structured like this in the first place.